src/vendor/OktaAuth.js

```diff
@@ -85,11 +85,13 @@ if (!Array.prototype.indexOf) {
   };
 }
 
-
-function isVerifySupported() {
-  return typeof crypto !== 'undefined' && crypto.subtle && typeof Uint8Array !== 'undefined';
+if (!Array.isArray) {
```

Comment:
Do we have a jira already for pulling in OktaAuth from npm? If not, can you create one? I'm assuming that this is dependent on us publishing to NPM.